### ISSUE TITLE
Re-add Nick.com video support

### ISSUE THREAD

**Issue by 1033Forest** [ytdl-org/youtube-dl#27470]
## Checklist

- [x] I'm reporting a new site support request
- [x] I've verified that I'm running youtube-dl version **2020.12.14**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that none of provided URLs violate any copyrights
- [x] I've searched the bugtracker for similar site support requests including closed ones

## Example URLs

- Single video: https://www.nick.com/episodes/ff16rh/all-that-1116-season-1-ep-16

## Description

Now that nick.com relaunched their ability to play episodes on a PC just today, please re-add support for this website.

**Comment by october262** [comment]
try using your TV provider log in credentials  -u  your user name &  -p your password 
see under Authentication Options: https://github.com/ytdl-org/youtube-dl

**Comment by 1033Forest** [comment]
Back in March-December 2018 I was able to download files from there without needing that.

**Comment by RuneMagle** [comment]
> 
> 
> Back in March-December 2018 I was able to download files from there without needing that.

True. And it's not even possible to use credentials as the site is not even support anymore.

```
C:\Users\Desktop\youtube-dl>youtube-dl.exe -F https://www.nick.com/episodes/b24gjl/kid-of-the-year-kid-of-the-year -v
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-F', 'https://www.nick.com/episodes/b24gjl/kid-of-the-year-kid-of-the-year', '-v']
[debug] Encodings: locale cp1252, fs mbcs, out cp850, pref cp1252
[debug] youtube-dl version 2020.11.26
[debug] Python version 3.4.4 (CPython) - Windows-10-10.0.18362
[debug] exe versions: ffmpeg git-2019-10-19-ed78ca4, ffprobe git-2019-10-19-ed78ca4, rtmpdump 2.4
[debug] Proxy map: {}
[generic] kid-of-the-year-kid-of-the-year: Requesting header
WARNING: Falling back on generic information extractor.
[generic] kid-of-the-year-kid-of-the-year: Downloading webpage
[generic] kid-of-the-year-kid-of-the-year: Extracting information
ERROR: Unsupported URL: https://www.nick.com/episodes/b24gjl/kid-of-the-year-kid-of-the-year
```

**Comment by 1033Forest** [comment]
Exactly. Which is why I'm asking if support for this site could be re-added.

**Comment by viceless** [comment]
This has still not been fixed. Please re-implement Nick.com it has undergone many changes and no longer functions with youtube-dl!

**Comment by Steveland** [comment]
Still does not work, I get ERROR: Unsupported URL for nick.com urls on latest youtube-dl version 2021.06.06

**Comment by bencollinz** [comment]
> Still does not work, I get ERROR: Unsupported URL for nick.com urls on latest youtube-dl version 2021.06.06

same. still.

**Comment by 7ngel** [comment]
> > Still does not work, I get ERROR: Unsupported URL for nick.com urls on latest youtube-dl version 2021.06.06
> 
> same. still.

Ya what's the point of these issue reports if nothing is ever fixed.
